core-api/src/main/java/com/sequenceiq/cloudbreak/api/model/CredentialResponse.java

```diff
@@ -16,6 +16,9 @@
     @ApiModelProperty(ModelDescriptions.PUBLIC_IN_ACCOUNT)
     private boolean publicInAccount = true;
 
+    @ApiModelProperty(ModelDescriptions.ATTRIBUTES)
+    private String attributes;
```

Comment:
For credential attributes some of the actual values needs to go back in the response so a separate filed should be used.